islandora_ocr.install

```diff
@@ -36,13 +36,19 @@ function islandora_ocr_enable() {
  */
 function islandora_ocr_cleanup_derivatives() {
   if (module_exists('islandora_book')) {
-    $book = variable_get('islandora_book_ingest_derivatives');
+    $book = variable_get('islandora_book_ingest_derivatives', array(
+      'pdf' => 'pdf',
+      'image' => 'image',
+      'ocr' => FALSE));
     $book['ocr'] = FALSE;
```

Comment:
Why is ocr being defaulted to FALSE and then set to FALSE again?

Reply:
Ok, checked code and I know why! but, as my 
https://github.com/Islandora/islandora_ocr/pull/95#discussion_r226773583 says this should be defaulted to `array('ocr')` to make sure code is more portable.